workflow-dispatch/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Workflow Dispatch
+description: Triggers a workflow with the workflow_dispatch event.
+inputs:
+  owner:
+    description: "Name of the repository owner in which the workflow resides. Defaults to the one who owns the workflow in which this action is running."
+    required: false
+  repo:
+    description: "Name of the repository in which the workflow resides. Defaults to the one in which the workflow currently running resides."
+    required: false
+  token:
+    description: "Github token. Cannot be ${{ secrets.GITHUB_TOKEN }} provided in the worflow run."
+    required: true
+  workflow:
+    description: "Name of the workflow file with the extension (ex: workflow.yml)."
+    required: true
+  ref:
+    description: "The reference of the workflow run. The reference can be a branch, tag, or a commit SHA. Defaults to master."
+    required: true
+    default: "master"
```

Comment:
I think that we should be more _forward thinking_ and change the default branch name to `main`.

More on why that change matter here: https://github.com/github/renaming